suites/apis/dataStageOIDCFlowTest.js

```diff
@@ -1,73 +1,76 @@
 /*
  This test plan has a few pre-requisites:
  1. Google & NIH accounts.
- 2. Access to the target commons environment (corresponding user entries in users.yaml - Google credential with "admin" access).
- 3. Existing files (successfully uploaded and indexed). A portion of these files must have a proper ACL configuration for both Google & NIH accounts, some files need to have an ACL versus Project Access mismatch to test "Access Denied" scenarios.
+ 2. Access to the target commons environment (corresponding user entries in users.yaml
+    - Google credential with "admin" access).
+ 3. Existing files (successfully uploaded and indexed). A portion of these files must have
+    a proper ACL configuration for both Google & NIH accounts, some files need to have an
+    ACL versus Project Access mismatch to test "Access Denied" scenarios.
  4. Client ID provided by developers (required for OIDC bootstrapping).
- 5. Client Secret provided by developers (Used for basic auth to obtain tokens and to refresh the access token).
+ 5. Client Secret provided by developers (Used for basic auth to obtain tokens and to
+    refresh the access token).
 */
-Feature('Testing OIDC flow and pre-signed URL to check tokens - https://ctds-planx.atlassian.net/browse/PXP-4649');
+Feature('Testing OIDC flow and pre-signed URL to check tokens - PXP-4649');
 
-// To be executed with GEN3_SKIP_PROJ_SETUP=true -- No need to set up program / retrieve access token, etc.
+// To be executed with GEN3_SKIP_PROJ_SETUP=true
+// No need to set up program / retrieve access token, etc.
 
-const readline = require('readline');
 const chai = require('chai');
-const user = require('../../utils/user.js');
 const fenceProps = require('../../services/apis/fence/fenceProps.js');
 const { interactive, ifInteractive } = require('../../utils/interactive.js');
 const { Gen3Response, getAccessTokenHeader, requestUserInput } = require('../../utils/apiUtil');
 
-const { expect } = chai;
+const { expect } = chai.expect; // eslint-disable-line no-redeclare
```

Comment:
needs to be changed to `const { expect }  = chai` OR `const { expect }  = require('chai');` after removing line 18

Reply:
Just pushed a commit to address that.
Waiting for PR check now...